Pocket Casts Watch App Extension/EpisodeView.swift

```diff
@@ -94,8 +94,8 @@ struct EpisodeView: View {
                 }
             }
 
-            if viewModel.supportsPodcastNavigation, let podcast = viewModel.parentPodcast, let viewModel = PodcastEpisodeListViewModel(podcast: podcast) {
-                NavigationLink(destination: PodcastEpisodeListView(viewModel: viewModel)) {
+            if viewModel.supportsPodcastNavigation, let podcast = viewModel.parentPodcast {
+                NavigationLink(destination: PodcastEpisodeListView(viewModel: .init(podcast: podcast))) {
```

Comment:
`let viewModel = PodcastEpisodeListViewModel(podcast: podcast)` did not produce an `Optional`. Xcode 14.3 doesn't allow this inconsistent syntax to build.